### PR TITLE
Replace install script with Makefile 

### PR DESCRIPTION
**Pull request by lvxnull**
This is better than setup.py and the install script in a few ways:
- building .mo files is separate from installation, which avoids possible permission issues when installing as root
- only recreates the .mo files when necessary
- supports arbitrary prefixes
- `glib-compile-schemas` being it's own target allows packaging tools to skip it

### CODE REVIEW

**Comment by Stunkymonkey** [comment]
is the goal here to replace the other install script?

File: nautilus_open_any_terminal/locale/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+LANGUAGES := de es fr pl pt_BR ru sw tr zh_CN
```
Comment (Stunkymonkey) [inline nautilus_open_any_terminal/locale/Makefile:1]: would it be possible to not have this hard-coded here? I will forget it for the next language.

**Comment by lvxnull** [comment]
> is the goal here to replace the other install script?
yes. 3 different build systems in a single project seems excessive.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+clean:
+	$(MAKE) -C $(LOCALES) clean
+
+install:
+	install -Dm644 $(EXTSRC) -t $(EXTDEST)
+	$(MAKE) -C $(LOCALES) install
+	install -Dm644 $(SCHEMASRC) -t $(SCHEMADEST)
+
+schema:
```
Comment (Stunkymonkey) [inline Makefile:26]: do we want to run this with `install` too?

Comment (lvxnull) [reply Makefile:26]: i want to be able to use `install` without compiling the schemas when building the package for this project. the user can still install and compile the schemas by using `make install schema`

**Review by Stunkymonkey — APPROVED** [review]

**Pull request merged**